_posts/2021-01-15-fault-tolerance-5-0.adoc

```diff
@@ -0,0 +1,168 @@
+:page-layout: post
+:page-title: Fault Tolerance 5.0
+:page-synopsis:  SmallRye Fault Tolerance 5.0 released, implementing MicroProfile Fault Tolerance 3.0 and bringing a lot of other improvements.
+:page-tags: [announcement, microprofile]
+:page-date: 2021-01-18 09:00:00.000 +0100
+:page-author: lthon
+:smallrye-ft: SmallRye Fault Tolerance
+:microprofile-ft: MicroProfile Fault Tolerance
+:uri-smallrye-ft-docs: https://smallrye.io/docs/smallrye-fault-tolerance/5.0.0
```

Comment:
Marked as draft because I need to verify if this value is correct. Which I can only after #41 is merged. Otherwise this is ready for review.

Reply:
Actually now I need #44 to be merged first :-)

Reply:
Why don't you join them into a single PR? :)

Reply:
Because I can't know the content of this variable before the other PR is merged and docs published. I can _guess_ (which I did), but want to verify.